docsrc/docextract_settings.rst

```diff
@@ -122,8 +122,8 @@ Force Render
 
 Setting::
 
-    "force_render": True
-    "force_render": False (default)
+    "force_render": True (default since IPA 4.7.0)
+    "force_render": False
```

Comment:
do we want to be more explicit that this is the default pre IPA 4.8?

Reply:
sure! can't hurt